### ISSUE TITLE
общие вопросы 

### ISSUE THREAD

**Issue by Anton-15** [Anton-15/-#11]

**Comment by Anton-15** [comment]
![image](https://user-images.githubusercontent.com/54960012/70472701-9bf8ed00-1ae0-11ea-931e-a3c476a53e99.png)
 правильно ли то что атрибуты таблицы у меня все на английском языке а сами таблицы на русском ?

**Comment by wldmr-key** [comment]
конечно, это не очень последовательно, но не греховно. В следующий раз будешь мудрее и все назовешь на одном языке (лучше английском - проще будет)

**Comment by Anton-15** [comment]
Что надо написать в записку в пункт "Проектно-технологическая часть"

**Comment by wldmr-key** [comment]
)) ты несогласен ни с чем из предложений методички?

**Comment by Anton-15** [comment]
у меня вышло ровно 20 страниц . Итоговую версию записки Вам скидывать ?

**Comment by Anton-15** [comment]
в презентацию для защиты надо логическую или концептуальную схему ? 
так же надо несколько скринов приложения ? 
и можно добавить схему алгоритма идентификации сотрудника ? 
что еще надо ?

**Comment by wldmr-key** [comment]
постановку задачи
и все это
спасибо за внимание ))
